### PR TITLE
Deplyment script update

### PR DESCRIPTION
**Pull request by olenovyk**
The commands.txt file contains a cli command to run the deployment script. You will need to add variables used in the command to your .env file and load them before running the script with source .env

### CODE REVIEW

File: src/Vault.sol [hunk 1]
```diff
@@ -20,25 +21,43 @@ contract Vault {
...
         beneficiaryAddress = _beneficiary;
         imETH = ImpactETHtoken(_imETHaddress);
+        isContractActive = true;
     }
 
     /**
         @notice This function allows users to deposit ETH to the contract. The amount of ETH deposited will be minted as imETH tokens
      */
     function deposit() external payable {
+        require(isContractActive, 'Contract is paused');
```
Comment (gtala) [inline src/Vault.sol]: this required could be a modifier, something like: **onlyActiveContract**

File: src/Vault.sol [hunk 2]
```diff
@@ -51,25 +70,53 @@ contract Vault {
...
+        totalDepositedEth -= _amountToWithdraw;
         stETH.transfer(msg.sender, stETH.getSharesByPooledEth(_amountToWithdraw));
+        emit Withdraw(msg.sender, _amountToWithdraw);
     }
 
     /**
         @notice This function allows users to stake available ETH in the contract with Lido
      */
     function stakeToLido() external payable {
+        require(isContractActive, 'Contract is paused');
```
Comment (gtala) [inline src/Vault.sol]: this required could be a modifier, something like: onlyActiveContract

**Review by gtala — CHANGES_REQUESTED** [review]
I Think it will be better implement Open Zeppelin Pausable  contract 


https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/security/Pausable.sol

We shouldn't reinvent the wheel

File: src/Vault.sol (hunk 1 above)
Comment (olenovyk) [reply src/Vault.sol]: Agreed. Just implemented Pausable instead, please review :)

**Review by gtala — APPROVED** [review]
LGTM!

**Pull request merged**